### PR TITLE
Feature/my wallet

### PR DESCRIPTION
**Pull request by saymon-develop**
- [Add transactions list](https://github.com/dexe-network/investing-dashboard/commit/e9a4204e4bafb6f58958a68a917a4a78ed57040d)
- [Refactoring GradientBorder](https://github.com/dexe-network/investing-dashboard/commit/3b9cda5d974f305637eadcf9248304d776b6ac09)

### CODE REVIEW

**Review by edward-arinin-web-dev — CHANGES_REQUESTED** [review]

File: src/components/TransactionHistory/index.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        clearTimeout(t)
+      }, 400)
+    }
+  }, [expanded, initialTop])
+
+  return (
+    <Container>
+      <Heading ref={titleRef}>Transactions History</Heading>
+      <Content
+        style={{
```
[the remark this answers is not in the document]
Comment (saymon-develop) [reply src/components/TransactionHistory/index.tsx]: Updated

File: src/components/TransactionHistory/index.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+          background: "linear-gradient(64.44deg, #0c1017 32.35%, #181d26 100%)",
+        }}
+        animate={expanded ? "visible" : "hidden"}
+        initial="hidden"
+        transition={{ duration: 0.4 }}
+        variants={{
```
[the remark this answers is not in the document]
Comment (saymon-develop) [reply src/components/TransactionHistory/index.tsx]: Moved variants to motion/variants.ts

File: src/components/TransactionHistory/index.tsx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+  return (
+    <Container>
+      <Heading ref={titleRef}>Transactions History</Heading>
+      <Content
+        style={{
+          background: "linear-gradient(64.44deg, #0c1017 32.35%, #181d26 100%)",
+        }}
+        animate={expanded ? "visible" : "hidden"}
+        initial="hidden"
+        transition={{ duration: 0.4 }}
```
[the remark this answers is not in the document]
Comment (saymon-develop) [reply src/components/TransactionHistory/index.tsx]: Duration has been reduced to 0.2

File: src/pages/Wallet/index.tsx [hunk 4]
```diff
@@ -256,7 +253,9 @@ export default function Wallet() {
 
   return (
     <>
-      <Header title={EHeaderTitles.myWallet} />
+      <Header>
+        {txListExpanded ? "Transactions History" : EHeaderTitles.myWallet}
+      </Header>
       <Container
         initial={{ opacity: 0, y: -15 }}
         animate={{ opacity: 1, y: 0 }}
```
[the remark this answers is not in the document]
Comment (saymon-develop) [reply src/pages/Wallet/index.tsx:261]: - removed background from expanded list
- overlayed elements hided when list is open

[the remark this answers is not in the document]
Comment (edward-arinin-web-dev) [reply src/pages/Wallet/index.tsx:261]: анимация топ 👍

**Review by edward-arinin-web-dev — APPROVED** [review]

**Pull request merged**